tests/std/tests/Dev11_0493504_error_category_lifetime/test.cpp

```diff
@@ -62,7 +78,67 @@ void test_gh_2893() {
         (make_error_condition(errc::no_such_file_or_directory) == error_code{ERROR_BAD_NET_NAME, system_category()}));
 }
 
+// Also test LWG-3629 make_error_code and make_error_condition are customization points
+namespace test_ns {
+    struct friendly_error {
+        friend error_code make_error_code(friendly_error) {
+            return error_code{};
+        }
+
+        friend error_condition make_error_condition(friendly_error) {
+            return error_condition{};
+        }
+    };
+
+    struct converted_errc : enable_if<false> {

```

Comment:
It took me an embarrassingly long time to see that we're inheriting from `enable_if` purely so this class will have `std` as an associated namespace. I think this is just subtle enough to deserve a comment.